includes/class-wcpdf-frontend.php

```diff
@@ -63,6 +64,28 @@ public function my_account_pdf_link( $actions, $order ) {
 		return apply_filters( 'wpo_wcpdf_myaccount_actions', $actions, $order );
 	}
 
+	/**
+	 * Open PDF on My Account page in a new browser tab/window
+	 */
+	public function open_my_account_pdf_link_on_new_tab() {
+		if ( function_exists( 'is_account_page' ) && is_account_page() ) {
+			if ( $general_settings = get_option( 'wpo_wcpdf_settings_general' ) ) {
+				if ( isset( $general_settings['download_display'] ) && $general_settings['download_display'] == 'display' ) {
+					wp_add_inline_script( 'jquery', "
+						jQuery( function( $ ) {
```

Comment:
While I think it's beneficial for the site speed to include this inline, for development purposes it's maybe nicer to keep it in a file. How about using `file_get_contents()` (or `ob_start()` + `include()` + `ob_get_clean()`) instead?

Reply:
Should be resolved in [this commit](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/pull/195/commits/23635ea03740cca971bbc070ecefb9f4f1f0f099).